test/testGraphical.m

```diff
@@ -0,0 +1,47 @@
+function [ status ] = testGraphical( varargin )
+%TESTGRAPHICAL Runs the M2T test suite to produce graphical output
+%
+% This is quite a thin wrapper around testMatlab2tikz to run the test suite to
+% produce a PDF side-by-side report.
+%
+% Its allowed arguments are the same as those of testMatlab2tikz.
+%
+% Usage:
+%
+%     status = testGraphical(...) % gives programmatical access to the data
+%
+%     testGraphical(...); % automatically invokes makeLatexReport afterwards
+%
+% See also: testMatlab2tikz, testHeadless, makeLatexReport
+
+    cwd = initializeWorkingDirectory();
```

Comment:
Given the discussion in #470, I'd say that we should not do it from within the test runners, but in a separate function (which is also where those directory changes should go).

Let's solve those issues in a new PR once this one is merged to avoid that this huge PR grows even bigger than it already is.